carma_wm_ctrl/src/main.cpp

```diff
@@ -14,16 +14,22 @@
  * the License.
  */
 
-#include <carma_wm_ctrl/WMBroadcasterNode.h>
-#include <ros/ros.h>
+#include <carma_wm_ctrl/WMBroadcasterNode.hpp>
+#include <rclcpp/rclcpp.hpp>
+
 // Main execution
 int main(int argc, char** argv)
 {
   // Initialize node
-  ros::init(argc, argv, "wm_broadcaster");
-  carma_wm_ctrl::WMBroadcasterNode node;
+  rclcpp::init(argc, argv);
+  
+  auto node = std::make_shared<carma_wm_ctrl::WMBroadcasterNode>(rclcpp::NodeOptions());
+
+  node->initializeWorker(node);
```

Comment:
How would this function get called when launched from a component?
This should probably be occurring automatically in handle_on_configure instead of here.

Reply:
deleted this since it is there in handle_on_configure